pyne/variancereduction.py

```diff
@@ -83,43 +83,45 @@ def cadis(adj_flux_mesh, adj_flux_tag, q_mesh, q_tag,
                                                                q_mesh, q_tag))
 
     # create volume element (ve) iterators
-    adj_ves = adj_flux_mesh.mesh.iterate(iBase.Type.region, iMesh.Topology.all)
-    q_ves = q_mesh.mesh.iterate(iBase.Type.region, iMesh.Topology.all)
+    adj_ves = mesh_iterate(adj_flux_mesh.mesh)
```

Comment:
It might be worth making `mesh_iterate` a method of the mesh class again. It definitely reads better that way I think. Not critical though by any means.

Reply:
probably not necessary now